### ISSUE TITLE
NTR: Maternal cat exposure (Gestation & perinatal history)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7873]
New term request Maternal cat exposure (segal_210928200158):   UMLS: 0238908

**Comment by pnrobinson** [comment]
@MickeySegal
Should this be Maternal household cat exposure or something like that to indicate repeated exposures? Is the intended context toxoplasmosis?

**Comment by MickeySegal** [comment]
Toxoplasmosis is the context.  Fine to specify household.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000784
- label: Maternal household cat exposure
- parents: [Pregnancy history (HP:0002686)](https://hpo.jax.org/app/browse/term/HP:0002686)
- pmid: 
- def: The mother of a fetus or index patient has a history of regular exposure to one or more cats, e.g., in the household.
- comment: This exposure may confer a risk for toxoplasmosis in the fetus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7873
- synonyms:
